src/storybook/story-to-test/index.ts

```diff
@@ -0,0 +1,59 @@
+import path from "path";
+import os from "os";
+import fs from "fs-extra";
+import _ from "lodash";
+import type { Test } from "hermione";
+import { writeStoryTestsFile } from "./write-tests-file";
+import { STORYBOOK_TEST_DIRNAME } from "../../constants";
+import type { StorybookStoryExtended } from "../get-stories";
+
+export type HermioneOpts = {
+    autoScreenshots: boolean;
+};
+
+const hermioneTestNameSuffix = ".hermione.js";
+
+const getStorybookTestsTmpDir = (): string => path.join(os.tmpdir(), STORYBOOK_TEST_DIRNAME);
+
+const getEmptyStorybookTestsTmpDir = async (): Promise<string> => {
+    const testsTmpDir = getStorybookTestsTmpDir();
+
+    await fs.emptyDir(testsTmpDir);
+
+    return testsTmpDir;
+};
+
+export const getStoryFile = (test: Test): string => {
```

Comment:
test.file = `<tmpDir>` + `"hermione-storybook-autogenerated"` + `<story.importPath>` + `".hermione.js"`

So if we strip away prefix and suffix, we end up with `story.importPath`, which is path to story, relative from project cwd